pkgs/top-level/python-packages.nix

```diff
@@ -9786,11 +9796,11 @@ in {
   py_scrypt = callPackage ../development/python-modules/py_scrypt/default.nix { };
 
   python_magic = buildPythonPackage rec {
-    name = "python-magic-0.4.10";
+    name = "python-magic-0.4.13";
 
     src = pkgs.fetchurl {
       url = "mirror://pypi/p/python-magic/${name}.tar.gz";
-      sha256 = "1hx2sjd4fdswswj3yydn2azxb59rjmi9b7jzh94lf1wnxijjizbr";
+      sha256 = "128j9y30zih6cyjyjnxhghnvpjm8vw40a1q7pgmrp035yvkaqkk0";
```

Comment:
Would you mind moving this to `development/python-modules/`, specifying `pname`, `version` and using `fetchPypi`?